dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/MaintenanceControllerTest.java

```diff
@@ -33,15 +33,15 @@
 
 import org.hisp.dhis.common.CodeGenerator;
 import org.hisp.dhis.test.web.HttpStatus;
-import org.hisp.dhis.test.webapi.H2ControllerIntegrationTestBase;
+import org.hisp.dhis.test.webapi.PostgresControllerIntegrationTestBase;
 import org.junit.jupiter.api.Test;
 
 /**
  * Tests {@link MaintenanceController} using (mocked) REST requests.
  *
  * @author Jan Bernitt
  */
-class MaintenanceControllerTest extends H2ControllerIntegrationTestBase {
+class MaintenanceControllerTest extends PostgresControllerIntegrationTestBase {
```

Comment:
Why is this change needed?

Reply:
It's actually not needed in this PR, but there's [this following PR](https://github.com/dhis2/dhis2-core/pull/18426) that continues the work on this topic, and it's needed there. Maybe we can leave it as it is, since it will be merged anyway.